Sources/Class-Package.php

```diff
@@ -793,13 +793,15 @@ public function connect($ftp_server, $ftp_port = 21, $ftp_user = 'anonymous', $f
 		if (!$this->connection)
 		{
 			$this->error = 'bad_server';
+            $this->last_message = 'Invalid Server';
```

Comment:
I really wish I knew what was wrong here.  It looks fine when I pull down a fresh copy of the repo. Seems to be some weird line ending stuff going on!